src/queries.js

```diff
@@ -2,25 +2,46 @@ import _ from 'lodash';
 
 // TODO: Sanitize string input of buckets
 
+// Helper function returning a query that defines a variable events_active
+// that can be used to filter away non-active time.
+// TODO: Handle events from all available browser buckets, as done in: https://github.com/ActivityWatch/aw-webui/pull/108
+// TODO: Don't count audible as activity if window isn't active
```

Comment:
This is a blocker. 

I think we would need to finish implementing the combination of window events and browser events (https://github.com/ActivityWatch/aw-server-rust/pull/179) before making headway on it.

Reply:
Nevermind, it works just fine as I did it in #262